plugins/hls-rename-plugin/src/Ide/Plugin/Rename.hs

```diff
@@ -187,8 +188,8 @@ refsAtName state nfp name = do
 
 nameLocs :: Name -> (HieAstResult, PositionMapping) -> [Location]
 nameLocs name (HAR _ _ rm _ _, pm) =
-    mapMaybe (toCurrentLocation pm . realSrcSpanToLocation . fst)
-             (concat $ M.lookup (Right name) rm)
+    concatMap (mapMaybe (toCurrentLocation pm . realSrcSpanToLocation . fst))
```

Comment:
... although here's another `mapMaybe`, so I guess we're already silently ignoring some things and maybe it's fine :shrug: